package.json

```diff
@@ -64,7 +64,7 @@
     "react-native": "0.73.6",
     "react-native-builder-bob": "^0.23.2",
     "release-it": "^14.2.2",
-    "typescript": "^4.5.2"
+    "typescript": "^5.8.3"
```

Comment:
First I planned to bump everything and even the example, but I am too tired and lazy. I don't even know if that works with 0.73 to be honest. It works with 0.77, 0.78 and 0.79, We should not support older versions anyway. I was *this* close to use MMKVs auto-install code, but that would've been to harsh for now.